web-services/boards.test.js

```diff
@@ -0,0 +1,11 @@
+var request = require('./request');
```

Comment:
This is an example of an integration test (versus a unit test). This phase was looking to "Research how to mock an API request in Jest" as it is very uncommon to have an integration test run as apart of the npm test cycle. Let's discuss next week.